recipes/go/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "go" %}
+{% set version = "1.9" %}
+{% set sha256 = "a4ab229028ed167ba1986825751463605264e44868362ca8e7accc8be057e993" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://storage.googleapis.com/golang/go1.9.src.tar.gz
```

Comment:
Would template out the version using Jinja in the URL.

Reply:
Fixed in the update.